### DISCUSSION TITLE
Support for multiple databases implemented

### DISCUSSION THREAD

**Discussion by Djamilson** [prisma/prisma#12804] [Q&A]
How would it look to work with two databases?

```prisma
datasource postgresDb {
  provider = "postgresql"
  url      = env("DATABASE_URL_POSTGRES")
}

datasource mongoDb {
  provider = "mongodb"
  url      = env("DATABASE_URL_MONGODB")
}

generator clientPostgres {
  provider = "prisma-client-js"
  output   = "./generated/postgres"
}

generator clientMongoDb {
  provider = "prisma-client-js"
  output   = "./generated/mongodb"
}
```

**Comment by austincrim** [comment]
Hey @Djamilson 👋,

Please take a look at the outlined solution in [this existing issue](https://github.com/prisma/prisma/issues/2443#issuecomment-630679118). In short, if you are using multiple different database providers, you will need multiple `schema.prisma` files and multiple generated Prisma Clients, similar to what you outlined above.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
